Failed to build pyopencl 2021.1 from source on macos X

Adding `"-arch", "arm64"` cxx flags (PR #405) makes build fail on macos X with errors like:
```
     In file included from src/wrap_constants.cpp:30:
    In file included from src/wrap_cl.hpp:54:
    In file included from /Library/Developer/CommandLineTools/SDKs/MacOSX10.15.sdk/System/Library/Frameworks/OpenCL.framework/Headers/opencl.h:12:
    In file included from /Library/Developer/CommandLineTools/SDKs/MacOSX10.15.sdk/System/Library/Frameworks/OpenCL.framework/Headers/cl.h:28:
    In file included from /Library/Developer/CommandLineTools/SDKs/MacOSX10.15.sdk/System/Library/Frameworks/OpenCL.framework/Headers/cl_platform.h:213:
    In file included from /Applications/Xcode_12.2.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/stdint.h:123:
    In file included from /Applications/Xcode_12.2.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/clang/12.0.0/include/stdint.h:52:
    In file included from /Library/Developer/CommandLineTools/SDKs/MacOSX10.15.sdk/usr/include/stdint.h:53:
    In file included from /Library/Developer/CommandLineTools/SDKs/MacOSX10.15.sdk/usr/include/sys/_types/_intptr_t.h:30:
    /Library/Developer/CommandLineTools/SDKs/MacOSX10.15.sdk/usr/include/machine/types.h:37:2: error: architecture not supported
    #error architecture not supported
```

--- a/setup.py
+++ b/setup.py
@@ -55,9 +55,7 @@ def get_config_schema():
 
         default_libs = []
         default_cxxflags = default_cxxflags + [
-                "-stdlib=libc++", "-mmacosx-version-min=10.7",
-                "-arch', 'i386", "-arch", "x86_64", "-arch", "arm64",
-                ]
+                "-stdlib=libc++", "-mmacosx-version-min=10.7"]
 
         from os.path import isdir
         for srp in sysroot_paths:

